languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -34459,6 +34476,82 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TYPOGRAPHIE" name="Typographie" type="typographical">
+      <rulegroup id="VIRGULE_TEMPS" name="virgule temps" default="temp_off">
+          <rule>
+              <pattern>
+                  <token postag="R pers suj.*" postag_regexp="yes"><exception regexp="yes">vous|nous</exception></token>
+                  <unify>
+                      <feature id="person"/>
+                      <feature id="tense"/>
+                      <marker>
+                          <token postag="V.*" postag_regexp="yes"/>
+                          <unify-ignore>
+                              <token><match no="0"/></token>
+                          </unify-ignore>
+                          <token postag="V.*" postag_regexp="yes"/>
+                      </marker>
+                  </unify>
+              </pattern>
+              <message>Une virgule semble être requise dans ce contexte.</message>
+              <suggestion>\1 \2, \3 \4</suggestion>
+              <example correction="monte, on descend">On <marker>monte on descend</marker>.</example>
+          </rule>
+          <rule>
+              <antipattern>
+                  <token postag="R pers suj.*" postag_regexp="yes"/>
+                  <token inflected="yes">prendre</token>
+                  <token postag="V.*" postag_regexp="yes"/>
+                  <token>et</token>
+                  <token postag="V.*" postag_regexp="yes"/>
+              </antipattern>
+              <pattern>
+                  <token postag="R pers suj.*" postag_regexp="yes"/>
+                  <unify>
+                      <feature id="person"/>
+                      <feature id="tense"/>
+                      <marker>
+                          <token postag="V.*" postag_regexp="yes">
+                              <exception postag="V etre.*" postag_regexp="yes"/>
+                              <exception postag="V avoir.*" postag_regexp="yes"/>
+                          </token>
+                          <token postag="V.*" postag_regexp="yes" regexp="yes">(?-i)[a-z].*<exception regexp="yes">casques?|confiance|enceinte</exception></token>
+                      </marker>
+                      <unify-ignore>
+                          <token>et</token>
+                      </unify-ignore>
+                      <token postag="V.*" postag_regexp="yes"/>
+                  </unify>
+              </pattern>
+              <message>Une virgule semble être requise dans ce contexte.</message>
+              <suggestion>\1 \2, \3</suggestion>
+              <example correction="monte, descend">Elle <marker>monte descend</marker> et recule.</example>
```

Comment:
Don't forget to run the maven test before pushing. Here the `correction` doesn't match the `suggestion`—thus, the test fails.

Reply:
This goes for all three subrules, btw.